package.json

```
@@ -59,7 +59,7 @@
     "sass": "^1.71.1",
     "stylelint": "^16.2.1",
     "stylelint-config-recommended-scss": "^14.0.0",
-    "typescript": "^5.3.3",
+    "typescript": "5.5",
```

This locks us to exactly version 5.5 of TypeScript, preventing us from receiving patch updates that may fix bugs before the next major release. Instead, we should tell npm to install the best version of TypeScript that has no breaking changes since 5.5:
```suggestion
    "typescript": "^5.5",
```